nano/node/nodeconfig.hpp

```diff
@@ -64,6 +64,8 @@ class node_config
 	std::chrono::seconds tcp_client_timeout{ std::chrono::seconds (5) };
 	std::chrono::seconds tcp_server_timeout{ std::chrono::seconds (30) };
 	std::chrono::nanoseconds pow_sleep_interval{ 0 };
+	/** Default maximum incoming TCP connections, including realtime network & bootstrap */
+	unsigned tcp_incoming_connections_max{ 256 };
```

Comment:
Since we lack an overlay network, is this too low?

Reply:
Updated to 1024